docs/server/source/data-models/transaction-model.rst

```diff
@@ -0,0 +1,62 @@
+.. raw:: html
+
+   <style>
+    .rst-content a.internal[href*='/schema/'] {
+        border: solid 1px #e1e4e5;
+        font-family: monospace;
+        font-size: 12px;
+        color: blue;
+        padding: 2px 4px;
+        background-color: white;
+    }
+   </style>
+
+=====================
+The Transaction Model
+=====================
+
+A transaction has the following structure:
+
+.. code-block:: json
+
+    {
+        "id": "<hash of transaction, excluding signatures (see explanation)>",
+        "version": "<version number of the transaction model>",
+        "transaction": {
+            "fulfillments": ["<list of fulfillments>"],
+            "conditions": ["<list of conditions>"],
+            "operation": "<string>",
+            "asset": "<digital asset description (explained in the next section)>",
+            "metadata": {
+                "id": "<uuid>",
+                "data": "<any JSON document>"
+            }
+        }
+    }
+
+Here's some explanation of the contents of a :ref:`transaction <transaction>`:
+
+- :ref:`id <transaction.id>`: The id of the transaction, and also the database primary key.
```

Comment:
Yes, I agree. I have some ideas to unify the model docs, so that we just have the yaml files and links to some supporting examples (possibly also generated). I think that's for another day though. Do you think we should remove these descriptions now?